app/routes/missions.fermi/route.mdx

```diff
@@ -36,6 +36,12 @@ import logo from './logo.svg'
 
 </div>
 
+**JSON-Serialized GCN Notices Types in GCN Kafka:**
+
+Fermi/GBM distributes JSON-serialized notices for the detection of gamma-ray transients on the GCN Kafka topic `gcn.notices.fermi.gbm`.
+
+For schemas and exmaples see [/schema/stable/gcn/notices/fermi/gbm](/schema/stable/gcn/notices/fermi/gbm) or use the [Schema Browser](/docs/schema/stable/gcn/notices/fermi/gbm/).
+
```

Comment:
```suggestion
</WithFeature>
```

Reply:
@dakota002 could you please check the feature flag add-ons?

Reply:
The idea is right, but this will probably fail when it runs the checks due to spacing. I would recommend that @shb46 add these tags themselves to just catch all of the autoformatting at once

Reply:
@dakota002: Do I just commmit the two @Vidushi-GitHub's suggested changes from above?  What about the "import at the top of the file"?

Reply:
@Vidushi-GitHub, @dakota002: Please check the changes I made to the file.